packages/labs/src/grpc_web/ts_proto_library.bzl

```diff
@@ -68,7 +68,10 @@ def _build_protoc_command(target, ctx):
     protoc_command += " --plugin=protoc-gen-grpc-web=%s" % (ctx.executable._protoc_gen_grpc_web.path)
 
     protoc_output_dir = ctx.var["BINDIR"]
-    protoc_command += " --grpc-web_out=import_style=commonjs+dts,mode=grpcweb:%s" % (protoc_output_dir)
+    wire_format = "grpcwebtext"
```

Comment:
why map from text -> grpcwebtest and binary -> grpcweb?
They may have chosen dumb names, but if we keep the same names as the underlying tool, it's easier to read their documentation and know what to put in a BUILD file.

Reply:
Updated to use their names :)